src/components/SubTypeAccordion/SubTypeAccordion.js

```diff
@@ -24,10 +24,12 @@ const SubTypeAccordion = ({
   resourceIds,
   activeCollectionId,
   subType,
+  index,
 }) => {
   const dataArray = [{ title: subType, content: resourceIds }];
+  const firstHeaderStyle = index === 0 ? styles.firstHeader : {};
```

Comment:
there's not alternative to this, to style 1st elements?

Reply:
I searched but wasn't able to find a more elegant solution.